tools/snp.sh

```diff
@@ -275,6 +284,12 @@ install_dependencies() {
   # Needed to build 6.11.0-rc3 SNP kernel on the host
   pip install tomli
   
+  # Needed to build Coconut-SVSM
+  if $SVSM; then
+    sudo apt install -y libcunit1 libcunit1-dev build-essential libclang-dev autoconf \
+      autoconf-archive pkg-config automake libssl-dev libc6-dev gcc-multilib binutils make musl musl-tools
+  fi
```

Comment:
Some of these are already being installed (build-essential, libssl-dev, pkg-config)